e2e/tests/08_ingressclass.bats

```diff
@@ -16,6 +16,7 @@ teardown() {
   delete_ingressclass custom || true
   delete_ingressclass external-lb || true
   delete_ingressclass internal-lb || true
+  delete_ingressclass internal-lb || nonallowed
```

Comment:
I guess a typo, should be `delete_ingressclass nonallowed || true`

Reply:
yep, fixed